configurations/perf/regression-nosqlbench-aws.yaml

```diff
@@ -0,0 +1,6 @@
+test_duration: 360
+
+n_monitor_nodes: 1
```

Comment:
This should be in the test yaml, because it doesn't change between configurations.

Reply:
Moved to test yaml